openshift/template.yaml

```diff
@@ -367,20 +371,20 @@ objects:
 - apiVersion: batch/v2alpha1
   kind: CronJob
   metadata:
-    name: graph-sync-job
+    name: graph-sync
```

Comment:
This name is used in sources, renaming this will cause the /sync endpoint to break. Why there is a need to rename it?